src/tests/ftest/osa/osa_online_reintegration.py

```diff
@@ -166,6 +182,14 @@ def run_online_reintegration_test(self, num_pool):
         # Exclude one rank : other than rank 0.
         rank = random.randint(1, exclude_servers)
 
+        # Start the daos_racer thread
+        daos_racer_thread = threading.Thread(target=self.daos_racer_thread,
+                                             kwargs={
+                                              "results": self.ds_racer_queue
+                                              })
```

Comment:
Optional suggestion to improve readability:
```suggestion
        kwargs = {"results": self.ds_racer_queue}
        daos_racer_thread = threading.Thread(
            target=self.daos_racer_thread, kwargs=kwargs)
```

Reply:
Good suggestion. Done. Thanks.